discord/eval.rb

```diff
@@ -1,7 +1,7 @@
 module Eval
   extend Discordrb::Commands::CommandContainer
 
-  command(:eval, description: "Evals.") do |event, *code|
+  command(:eval, description: "Evals given input through the bot.") do |event, *code|
```

Comment:
Not at all what I told you to put

Reply:
How would you prefer I write it?  I don't use eval that much, so I'm glad to have your input.

Reply:
My mistake, I never specified. Put "Evaluates Ruby expressions." Then add something about "for Ruby experts, use with caution" or something